### PR TITLE
Rating history graps in user profile

### PR DESCRIPTION
**Pull request by jlelic**
- Added rating history graph to user profile.
- Rewrote the two components I touched to functional components. It's not exactly consistent with the rest of the application but it's better ¯\\_(ツ)_/¯

### CODE REVIEW

File: frontend/src/app/modules/matches/matches-computations.js [hunk 1]
```diff
@@ -15,6 +15,23 @@ export const generateMatchRatingChanges = (userId, userMatches) => userMatches.m
     }
 })
 
+export const generateRatingHistoryGraphForUser = (userMatches, userId, initialRating) => userMatches
```
Comment (jlelic) [inline frontend/src/app/modules/matches/matches-computations.js]: I think `generateRatingHistoryGraphForUser` may be too long 🤔

Comment (petrk-salsita) [reply frontend/src/app/modules/matches/matches-computations.js]: I think it's perfectly fine and descriptive

File: frontend/src/app/pages/Profile/index.js [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
   render() {
     if (!this.props.user) {
       return <div />
     }
 
-    return(
+    const {
+      user: { name, rating, id },
+      statistics: { totalMatches, winRatio, longestStreak, matchChanges }
+    } = this.props
```
Comment (petrk-salsita) [inline frontend/src/app/pages/Profile/index.js]: Some of the variables from the deconstruction are not used, right?

**Comment by petrk-salsita** [comment]
What an amazing piece of software architecture, thank you!

File: frontend/src/app/pages/Profile/index.js (hunk 2 above)
Comment (jlelic) [reply frontend/src/app/pages/Profile/index.js]: Nope, all of them are used.

**Comment by jlelic** [comment]
It's not every day you get to see a stateful `map` :genius:

**Pull request merged**